Userland/Libraries/LibWeb/DOMURL/URLSearchParams.cpp

```diff
@@ -56,11 +63,11 @@ ErrorOr<String> url_encode(Vector<QueryParam> const& tuples, StringView encoding
 
         // 2. Let name be the result of running percent-encode after encoding with encoding, tuple’s name, the application/x-www-form-urlencoded percent-encode set, and true.
         // FIXME: URL::Parser does not currently implement encoding.
-        auto name = TRY(URL::Parser::percent_encode_after_encoding(tuple.name, URL::PercentEncodeSet::ApplicationXWWWFormUrlencoded, true));
+        auto name = TRY(URL::Parser::percent_encode_after_encoding(*encoder, tuple.name, URL::PercentEncodeSet::ApplicationXWWWFormUrlencoded, true));
 
         // 3. Let value be the result of running percent-encode after encoding with encoding, tuple’s value, the application/x-www-form-urlencoded percent-encode set, and true.
         // FIXME: URL::Parser does not currently implement encoding.
```

Comment:
Can remove the FIXME, an the one above

Reply:
Thanks for reviewing. I've removed these FIXMEss in the latest revision.

Reply:
phone posting since my laptop is borked right now, so sorry if this isnt correct - but I think you may have accidentally undone these changes when rebasing. nice changes though!

Reply:
You're right, I have. Thanks for spotting that!